hudi-common/src/test/java/org/apache/hudi/common/testutils/HoodieTestTable.java

```diff
@@ -131,6 +131,7 @@ protected HoodieTestTable(String basePath, FileSystem fs, HoodieTableMetaClient
     this.basePath = basePath;
     this.fs = fs;
     this.metaClient = metaClient;
+    testTableState = HoodieTestTableState.of();
```

Comment:
Where is this used?

Reply:
The other static method in this class `HoodieTestTable#of` is already setting the `testTableState`. Just that it is missing in the constructor. This `state` member is used for local book keeping of the partition files, commits, etc., Only for the test usage.